src/components/Picture.astro

```diff
@@ -0,0 +1,30 @@
+---
+type Props = {
+  className?: string
+  src: string
+  alt: string
+}
+
+const medias = [
```

Comment:
It would be nice if we can also have a prop here to add a custom srcsize if we need to, maybe for larger screens 1600px (just an example). Let me know if that makes sense.